### PR TITLE
Remove jsonfile

### LINKED ISSUE #5: Drop jsonfile usage.
Lets consider dropping the `jsonfile` module. 

At this time it appears that we're only using it to read configuration. We've had great success with [config](https://www.npmjs.com/package/config) in the past. Additionally if you are looking to actually just read the JSON directly, you can require it, although in a slightly different format. 

`config.js` ->

``` javajscript
module.exports = {
  'myKey': 'myValue'
}
```

### PR DESCRIPTION
**Pull request by mattgrill**
Fixes #5

Signed-off-by: Matthew Grill matt@fourkitchens.com

### CODE REVIEW

**Comment by patrickocoffeyo** [comment]
This is great, however loading the json file is unnesseccary. When Aquifer initializes, it loads it's own config, validates it, and assigns it to a property of itself, since config is such an important part of Aquifer.

Config properties should be available here:
`Aquifer.project.config`

**Comment by patrickocoffeyo** [comment]
See 
- https://github.com/aquifer/aquifer/blob/master/lib/init.api.js#L32
- https://github.com/aquifer/aquifer/blob/master/lib/project.api.js#L39

**Comment by mattgrill** [comment]
@patrickocoffeyo If that's the case. I can just revise this to actually remove the config loading completely.

**Comment by mattgrill** [comment]
Config should be available at, `Aquifer.project.config.paths.make`

**Comment by patrickocoffeyo** [comment]
@mattgrill I think we can safely do that! Thank you for jumping onto this

File: bin/index.js [hunk 1]
```diff
@@ -57,9 +56,7 @@ module.exports = function(Aquifer, AquiferGitConfig) {
       return;
     }
 
-    var jsonPath        = path.join(Aquifer.projectDir, 'aquifer.json'),
-        json            = jsonFile.readFileSync(jsonPath),
-        make            = path.join(Aquifer.projectDir, json.paths.make),
+    var make            = path.join(Aquifer.projectDir, Aquifer.project.config.paths.make),
```
Comment (patrickocoffeyo) [inline bin/index.js]: I hate to be picky, but there is also a dynamically generated property on Aquifer that holds absolute paths to assets.

`path.join(Aquifer.projectDir, Aquifer.project.config.paths.make)`
can be
`Aquifer.project.absolutePaths.make`

**Comment by chasingmaxwell** [comment]
Yeah there's actually an issue for that: #4

**Comment by chasingmaxwell** [comment]
Looks good to me!

**Pull request merged**